### PR TITLE
Updated generateurl validation tests

### PR DESCRIPTION
**Pull request by karthikmsqa**
# Description

This PR includes the following proposed change(s):

- Completes https://justice.gov.bc.ca/jira/browse/FLA-153
- Refactored featured file and request calls to address the changes

## Type of change

- [x] New tests(non-breaking change which adds more tests)
- [x] Refactoring 
if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Tested locally

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/requestbuilders/GenerateUrlRequestBuilders.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+        request = given().spec(TestUtil.requestSpecification()).header(X_AUTH_USER_ID,validExistingCSOGuid).body(payloadData.generateUrlFinalPayload());
+        return request.when().post(resourceAPI.getResource()).then().spec(TestUtil.responseSpecification()).extract().response();
+    }
+
+    public Response requestWithNonExistingCSOAccountGuid(String resourceValue) throws IOException {
+        payloadData = new GenerateUrlPayload();
+        APIResources resourceAPI = APIResources.valueOf(resourceValue);
+
+        String  nonExistingCSOGuid = JsonDataReader.getCsoAccountGuid().getNonExistingCSOGuid();
```
Comment (sdevalapurkar-bcgov) [inline tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/requestbuilders/GenerateUrlRequestBuilders.java:33]: nit/question: looks like there is an extra space here after String
might just be github tho :)

File: tests/src/test/java/stepDefinitions/frontendstepdefinitions/EFileSubmissionTest.java [hunk 2]
```diff
@@ -12,6 +12,7 @@
 import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
 import org.junit.Assert;
+
```
Comment (sdevalapurkar-bcgov) [inline tests/src/test/java/stepDefinitions/frontendstepdefinitions/EFileSubmissionTest.java:15]: nit: can probably remove this and reduce number of files modified by 1 😄

File: tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/requestbuilders/GenerateUrlRequestBuilders.java (hunk 1 above)
Comment (karthikmsqa) [reply tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/requestbuilders/GenerateUrlRequestBuilders.java:33]: Good eyes!fixed. Thanks.

File: tests/src/test/java/stepDefinitions/frontendstepdefinitions/EFileSubmissionTest.java (hunk 2 above)
Comment (karthikmsqa) [reply tests/src/test/java/stepDefinitions/frontendstepdefinitions/EFileSubmissionTest.java:15]: fixed. thanks.

**Pull request merged**